CONTRIBUTING.md

```diff
@@ -142,20 +142,20 @@ tracing) enabled.
 The setup for a local rustc works as follows:
 ```sh
 # Clone the rust-lang/rust repo.
-git clone https://github.com/rust-lang/rust/ rustc
+git clone https://github.com/rust-lang/rust rustc
 cd rustc
 cp config.toml.example config.toml
 # Now edit `config.toml` and set `debug-assertions = true`.
 
-# Build a stage 2 rustc.
+# Build a stage 1 rustc.
 # This step can take 30 minutes or more.
-./x.py build --stage 2 compiler/rustc
+./x.py build --stage 1 compiler/rustc
```

Comment:
Still not sure why `compiler/rustc` is necessary, but that's a mystery for another day.